public/app/plugins/panel/multistat/types.ts

```diff
@@ -0,0 +1,66 @@
+import { PanelModel } from 'app/features/dashboard/panel_model';
+import { ThresholdModel } from './components/ThresholdManager/ThresholdEditor';
+
+export interface SeriesData {
+  datapoints: any[];
+  target: string;
+}
+
+export type DataList = SeriesData[];
+
+export interface SeriesStat {
+  alias?: string;
+  label?: string;
+  value?: number;
+  valueRounded?: number;
+  valueFormatted?: string;
+  flotpairs?: any[];
+  scopedVars?: any;
+}
+
+export interface MultistatPanelSize {
+  w: number;
+  h: number;
+}
+
+export interface BasePanelOptions {
+  links?: any[];
+}
+
+export interface MetricPanelOptions extends BasePanelOptions {
+  datasource?: any;
+  maxDataPoints?: number;
+  interval?: any;
+  targets?: any[];
+  cacheTimeout?: any;
+}
+
+export interface MultistatPanelOptions extends MetricPanelOptions {
+  layout?: MultistatPanelLayout;
+  viewMode?: MultistatPanelViewMode;
+  format?: any;
+  mappingType?: any;
+  nullPointMode?: any;
+  valueName?: string;
+  prefixFontSize?: string;
```

Comment:
do we need these font size options? It might be easier if Grafana could adjust and handle this automatically. But not sure.